integration/spark-datasource/src/test/scala/org/apache/spark/sql/carbondata/datasource/SparkCarbonDataSourceBinaryTest.scala

```diff
@@ -257,9 +255,9 @@ class SparkCarbonDataSourceBinaryTest extends FunSuite with BeforeAndAfterAll {
 
     test("insert into for hive and carbon, CTAS") {
         sql("DROP TABLE IF EXISTS hiveTable")
-        sql("DROP TABLE IF EXISTS carbontable")
```

Comment:
how this table name change is related to this PR , and why this is done??

Reply:
sometimes there are some exception because local metastore/warehouse.